components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -425,28 +623,68 @@ struct SignMessageRequest {
   string message;
 };
 
+// For default wallet setting, user assets, permission checks, importing
+// functionality and much more.
 interface BraveWalletService {
+  // Adds an observer for BraveWalletService
   AddObserver(pending_remote<BraveWalletServiceObserver> observer);
+
+  // Obtains the user assets for the specified chain ID
   GetUserAssets(string chain_id) => (array<ERCToken> tokens);
+
+  // Adds a user asset for the specified chain ID
   AddUserAsset(ERCToken token, string chain_id) => (bool success);
+
+  // Removes a user asset for the specified chain ID
   RemoveUserAsset(ERCToken token, string chain_id) => (bool success);
+
+  // Sets whether or not an asset is visible
```

Comment:
nit: `Sets whether or not an asset is visible for the specified chain ID` to be consistent.

Reply:
Updated